validation/src/test/scala/hmda/validation/rules/lar/validity/V570Spec.scala

```diff
@@ -0,0 +1,47 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.validation.dsl.Success
+
+class V570Spec extends RateSpreadEditCheckSpec {
+
+  override val check = V570
+
+  property("succeeds if lien status is 1 and rate spread is NA") {
+    succeedsWhen(lienStatus = 1, rateSpread = "NA")
+  }
+
+  property("fails if lien status = 1 and rate spread is < 1.5%") {
```

Comment:
oops, yes. For V575, that case is covered in among the other badly-formatted ones (because anything other than NN.NN is considered badly-formatted by other rules), but I didn't do that here. Will do.